bookclub-frontend/src/App.js

```diff
@@ -4,19 +4,27 @@ import Login from './components/sessions/Login';
 import { BrowserRouter as Router, Switch, Route } from 'react-router-dom';
 import { Button } from 'react-bootstrap';
 import ListReadBooks from './components/bookclub/ListReadBooks';
+import { useCookies } from 'react-cookie';
 import AddBook from './components/bookclub/AddBook';
 import CreateNewUser from './components/sessions/CreateNewUser';
 
 import './App.css';
 
 function App() {
   const [isLoggedIn, setIsLoggedIn] = useState(false);
+  const [uidCookie, setUidCookie, removeUidCookie] = useCookies(['UID']);
```

Comment:
I'm noticing that you have the same react hooks for AddBook, ListReadBooks, CreateNewUser and Login. Logistically, to make sure all four files are referencing to the same state as the one in `App.js`, you'd keep the hooks here, but pass them as props to the other four files (along with their methods `set...`/`remove...`)

The alternative to passing props down is to `useContext`, which would allow you to refer to the same `uidCookie`/`isLoggedIn` across all components. 

https://stackoverflow.com/questions/53451584/is-it-possible-to-share-states-between-components-using-the-usestate-hook-in-r